docs/src/components/indices/index.md

```diff
@@ -0,0 +1,8 @@
+# Indices
+
+Indices are the one of the core building blocks of the Fuel indexer service. Indices allow index operators and dApp authors to index specific blockchain into a database.
```

Comment:
"...index specific blockchain into a database."

Should we say "blockchain events" or another similar phrase?